### ISSUE TITLE
ohmGraphite sends float with comma instead of dots

### ISSUE THREAD

**Issue by makefu** [nickbabcock/OhmGraphite#5]
Hey,

i tried your project and so far really like it. Unfortunately it seems that due to my locale the floating points are being sent as comma instead of a dot. This seems to be due to the german locale and the automatic formatting of `data.Value` ( https://github.com/nickbabcock/OhmGraphite/blob/master/OhmGraphite/MetricTimer.cs#L88 ). It would be fantastic if you patch the code to always format float with dots instead of comma to avoid any locale issues. I'd fix the issue myself but unfortunately i do not have any experience with C#.

This is the log of the carbonCache:
```
May 03 22:19:36 wolf carbon-cache-a[633]: [listener] invalid line received from client 10.42.27.207:50415, ignoring [ohm.KAFFEEHALTER.hdd.0.factor.writeamplification 2,228571 1525378776]
```

Thanks in advance!

**Comment by nickbabcock** [comment]
Oof such a silly oversight 😊 

I'll have a fix out here shortly.

**Comment by nickbabcock** [comment]
Closed by 1e7704acf2118c

Released in [v0.2.1](https://github.com/nickbabcock/OhmGraphite/releases/tag/v0.2.1)

**Comment by makefu** [comment]
so fast, thanks!
